jetty-server/src/main/config/etc/jetty.xml

```diff
@@ -66,7 +66,7 @@
       <Set name="requestHeaderSize" property="jetty.httpConfig.requestHeaderSize"/>
       <Set name="responseHeaderSize" property="jetty.httpConfig.responseHeaderSize"/>
       <Set name="sendServerVersion" property="jetty.httpConfig.sendServerVersion"/>
-      <Set name="sendDateHeader" property="jetty.httpConfig.sendDateHeader"/>
+      <Set name="sendDateHeader"><Property name="jetty.httpConfig.sendDateHeader" default="false"/></Set>
```

Comment:
Another behavior change? Why?

Reply:
@sbordet I don't think you've understood the fundamental of this PR :)  The whole point is that commit https://github.com/eclipse/jetty.project/commit/343cf732b9a5aacd4c84fcb937a4b1624a1a0afd accidentally changed the configuration by removing the `default` values, so this PR is putting them back the way they were.